gdebug/docker/start_docker.sh

```diff
@@ -12,27 +12,20 @@ fi
 
 readonly DOCKER_DIR="$(cd "$(dirname "$0")" && pwd)"
 
-TEMP=$(mktemp -p $DOCKER_DIR --suffix=.yaml)
-function finish {
-  rm "$TEMP"
-}
-trap finish EXIT
-
 readonly ENVOY_PORT=$1
 readonly GRPC_ADDR=$2
 readonly GRPC_PORT=$3
 readonly ASSETS_PORT="8080"
 
-cat > "$TEMP" <<TERMINATOR
-version: '2'
+readonly CONFIG="version: '2'
 services:
   front-envoy:
     build:
       context: ./envoy/
       dockerfile: Dockerfile
     ports:
       # Forward host port to docker
-      - "$ENVOY_PORT:$ENVOY_PORT"
+      - '$ENVOY_PORT:$ENVOY_PORT'
```

Comment:
Does this work?

Reply:
It starts up ok and the echoing the var looks sane:
```
services:
  front-envoy:
    build:
      context: ./envoy/
      dockerfile: Dockerfile
    ports:
      # Forward host port to docker
      - '\''9900:9900'\''
    environment:
      - GRPC_HOST=127.0.0.1
      - GRPC_PORT=50051
      # The docker port for envoy
      - ENVOY_PORT=9900
      # The host+port for the static assets server
      - ASSETS_HOST=127.0.0.1
      - ASSETS_PORT=8080
    mem_limit: 1000000000
    network_mode: '\''host'\''
  back-static-assets:
    build:
      context: ./static-assets/
      dockerfile: Dockerfile
    environment:
      - PORT=8080
    ports:
      # Make the port availble to linked dockers but no need
      # to forward to host machine.
      - '\''8080:8080'\''
    mem_limit: 1000000000'
```

Reply:
This is failing for me on macOS (notice missing ports on `docker_front-envoy`):

``` console
❯ docker ps
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS                    NAMES
71e52171a59b        docker_front-envoy          "/bin/sh -c '/zprox/…"   10 minutes ago      Up 10 minutes                                docker_front-envoy_1
7aa410822bb9        docker_back-static-assets   "/bin/sh -c 'go run …"   10 minutes ago      Up 10 minutes       0.0.0.0:8080->8080/tcp   docker_back-static-assets_1
```

``` console
❯ docker-compose version
docker-compose version 1.23.2, build 1110ad01
docker-py version: 3.6.0
CPython version: 3.6.6
OpenSSL version: OpenSSL 1.1.0h  27 Mar 2018

❯ docker version
Client: Docker Engine - Community
 Version:           18.09.2
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        6247962
 Built:             Sun Feb 10 04:12:39 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       6247962
  Built:            Sun Feb 10 04:13:06 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```